projects/plugins/jetpack/extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -104,7 +104,7 @@ function render_block( $attributes, $content ) {
 		return render_error( __( 'Your podcast URL is invalid and couldn\'t be embedded. Please double check your URL.', 'jetpack' ) );
 	}
 
-	if ( isset( $attributes['selectedEpisodes'] ) && count( $attributes['selectedEpisodes'] ) ) {
+	if ( isset( $attributes['selectedEpisodes'] ) && is_countable( $attributes['selectedEpisodes'] ) && count( $attributes['selectedEpisodes'] ) ) {
```

Comment:
Testing `count( $attributes['selectedEpisodes'] )` is basically checking for it not being an empty array (or for being a non-array in PHP 7, but in that case the `array_map()` just below would break). So we could simplify this:
```suggestion
	if ( ! empty( $attributes['selectedEpisodes'] ) ) {
```